manifests/cluster-set.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: cluster.open-cluster-management.io/v1beta1
+apiVersion: cluster.open-cluster-management.io/v1beta2
```

Comment:
Note that v1beta2 is only from ACM 2.7.0 (for 2.6 it is still v1beta1)

Reply:
Yes, I know